test_mcp_debug_simple.py

```
@@ -1,4 +1,5 @@
 #!/usr/bin/env python3
+from config.mcp_config import MCPConfig
```

Ruff (0.11.9)

`config.mcp_config.MCPConfig` imported but unused

Remove unused import: `config.mcp_config.MCPConfig`

(F401)